src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,212 @@
+                + "JOIN cars_drivers ON cars.id = cars_drivers.car_id "
+                + "JOIN manufacturers ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars_drivers.driver_id = ? AND cars.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                Car car = getCar(resultSet);
+                car.setDrivers(getDrivers(car.getId()));
+                cars.add(car);
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all cars by driver id "
+                    + driverId, e);
+        }
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        car.setManufacturer(getManufacturer(resultSet));
+        return car;
+    }
+
+    private Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("manufacturer_id", Long.class);
+        String name = resultSet.getString("name");
+        String country = resultSet.getString("country");
+        return new Manufacturer(id, name, country);
+    }
+
+    private Driver getDriver(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("id", Long.class);
+        String name = resultSet.getString("name");
+        String licenseNumber = resultSet.getString("license_number");
+        return new Driver(id, name, licenseNumber);
+    }
+
+    private List<Driver> getDrivers(Long carId) {
+        String query = "SELECT id, name, license_number "
+                + "FROM drivers "
+                + "JOIN cars_drivers ON cars_drivers.driver_id = drivers.id "
+                + "WHERE cars_drivers.car_id = ? AND drivers.is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Couldn't get drivers by car's id " + carId,
+                    throwables);
+        }
+    }
+
+    private void addCarsDrivers(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?);";
```

Comment:
```suggestion
        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?)";
```
; - is not important here